### PR TITLE
feat(atlantis-chart): Helm best practice

### PR DESCRIPTION
**Pull request by tvarade**
Restart atlantis pod after updating any external configuration provided

### CODE REVIEW

File: charts/atlantis/templates/statefulset.yaml [hunk 1]
```diff
@@ -29,6 +29,7 @@ spec:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap-config.yaml") . | sha256sum }}
         checksum/repo-config: {{ include (print $.Template.BasePath "/configmap-repo-config.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/extra-manifests.yaml") . | sha256sum }}
```
Comment (lkysow) [inline charts/atlantis/templates/statefulset.yaml:32]: this is the same key as above...

Comment (mubarak-j) [inline charts/atlantis/templates/statefulset.yaml:32]:
```suggestion
        checksum/extra-manifests: {{ include (print $.Template.BasePath "/extra-manifests.yaml") . | sha256sum }}
```
use a unique name for annotations key

**Comment by tvarade** [comment]
@lkysow  any updates on this?

**Comment by lkysow** [comment]
@tvarade please addess my comment

**Comment by jamengual** [comment]
@tvarade willing to update the PR?

**Comment by nitrocode** [comment]
Couldnt you use staker's reloader to restart the pod if a dependent configuration is updated?

**Pull request closed without merging**